templates/profile.html

```diff
@@ -4,12 +4,18 @@
   var user_data={% raw user_info_json %};
   var routesObject = {
       surveys: "/api/surveys",
-      response: "/api/response"
+      response: "/api/response",
```

Comment:
Do we need the surveys/response here? I thought this page will just include groups/subscribe.